.travis.yml

```diff
@@ -42,7 +42,7 @@ jobs:
   include:
     - name: pylint tests
       script:
-        - pip install requests xxhash grpcio pyxattr kubernetes
+        - pip install requests xxhash grpcio pyxattr kubernetes pythoncli
```

Comment:
is it `glustercli` ?

Reply:
Doh. Yes. Travis didn't seem too upset by me typing rubbish though.